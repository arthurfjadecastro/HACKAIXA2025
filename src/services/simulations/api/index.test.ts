import * as simulationsApiIndex from './index';

describe('Simulations API Index', () => {
  it('should export simulations API modules', () => {
    expect(simulationsApiIndex).toBeDefined();
    expect(typeof simulationsApiIndex).toBe('object');
  });

  it('should re-export API functions without errors', () => {
    // Verifica se mÃ³dulos foram exportados sem erros
    expect(() => {
      const exported = { ...simulationsApiIndex };
      expect(exported).toBeDefined();
    }).not.toThrow();
  });

  it('should match snapshot', () => {
    expect(Object.keys(simulationsApiIndex).sort()).toMatchSnapshot();
  });
});
