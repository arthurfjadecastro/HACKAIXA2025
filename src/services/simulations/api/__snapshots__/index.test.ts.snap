// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simulations API Index should match snapshot 1`] = `
[
  "simulationsApi",
]
`;
